data/deduplicator.go

```diff
@@ -31,14 +34,19 @@ func (d *DeduplicatorDescriptor) IsRegisteredWithAnyDeduplicator() bool {
 }
 
 func (d *DeduplicatorDescriptor) IsRegisteredWithNamedDeduplicator(name string) bool {
-	return d.Name == name
+	// TODO: Backwards compatible until after data migration to update deduplicator name scheme
+	return d.Name == name || d.Name == strings.TrimPrefix(name, "org.tidepool.")
```

Comment:
is it worth having `org.tidepool.` as a well known const? I can see we only use it once at the moment but just thinking...